web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -0,0 +1,67 @@
+# PDEP-10: PyArrow as a required dependency
+
+- Created: 17 April 2023
+- Status: Under discussion
+- Discussion: [#52711](https://github.com/pandas-dev/pandas/pull/52711)
+              [#52509](https://github.com/pandas-dev/pandas/issues/52509)
+- Author: [Matthew Roeschke](https://github.com/mroeschke)
+- Revision: 1
+
+## Abstract
+
+This PDEP proposes that:
+
+- PyArrow becomes a runtime dependency starting pandas 2.1
+- The minimum version of PyArrow supported starting pandas 2.1 is version 7.
+- The minimum version of PyArrow will be bumped every major pandas release to the highest
+  PyArrow version that has been released for at least 2 years, and the minimum PyArrow version will be
+  maintained for every minor version in the major version series.
+
+## Background
+
+PyArrow is an optional dependency of pandas that provides a wide range of supplemental features to pandas:
+
+- Since pandas 0.21.0, PyArrow provided I/O reading functionality for Parquet
+- Since pandas 1.2.0, pandas integrated PyArrow into the `ExtensionArray` interface to provide an optional string data type backed by PyArrow
+- Since pandas 1.4.0, PyArrow provided I/0 reading functionality for CSV
+- Since pandas 1.5.0, pandas provided an `ArrowExtensionArray` and `ArrowDtype` to support all PyArrow data types within the `ExtensionArray` interface
+- Since pandas 2.0.0, All I/O readers have the option to return PyArrow-backed data types, and many methods now utilize PyArrow compute functions to
+accelerate PyArrow-backed data in pandas, notibly string and datetime types.
+
+As of pandas 2.0, one can feasibly utilize PyArrow as an alternative data representation to NumPy with advantages such as:
+
+1. Consistent ``NA`` support for all data types
+2. Broader support of data types such as ``decimal``, ``date`` and nested types
```

Comment:
I think that in https://github.com/pandas-dev/pandas/issues/52429#issuecomment-1509395487, @jbrockmendel showed an example where performance can be better with string types.  Ideally, we'd have more examples like that, so probably adding a third item here would be worthwhile.

Reply:
https://phofl.github.io/pydata-berlin/pydata-berlin-2023/intro.html#70

and the following slides.